README.rst

```diff
@@ -72,6 +72,11 @@ Slack: `#go on Bazel Slack`_, `#bazel on Go Slack`_
 
 Announcements
 -------------
+2022-09-11
+  Release
+  `v0.35.0 <https://github.com/bazelbuild/rules_go/releases/tag/v0.35.0>`_
+  is now available. This release enhances support for bzlmod and gopackagesdriver.
```

Comment:
I think we should mention the M1 toolchain improvements as well as the increased minimum version here.